landing-page/types/headnode/content/styles/login.css

```diff
@@ -0,0 +1 @@
+/code/flight-webapp-components/builder/build/static/css/main.css
```

Comment:
This isn't needed and is breaking the omnibus builder.  See comments on https://github.com/openflighthpc/flight-webapp-components/pull/8

Reply:
The symbolic link itself isn't needed?